app/javascript/src/components/Dashboard/Notes/List/Card/Footer.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { Clock } from "neetoicons";
+import { Typography, Tooltip, Avatar, Tag } from "neetoui";
+import { useTranslation } from "react-i18next";
+
+import { USER_AVATAR_URL } from "components/Dashboard/Notes/constants";
+import {
+  getElapsedTime,
+  getDateString,
+} from "components/Dashboard/Notes/utils";
+
+const Footer = ({ createdAt }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="flex items-center justify-between">
+      <Tag label={t("common.gettingStarted")} style="secondary" type="solid" />
+      <div className="flex space-x-2">
+        <Tooltip content={getDateString(createdAt)} placement="bottom-start">
```

Comment:
Let's name this `formatDateString` cause we aren't just getting a date string which is already existing, but rather we are formatting current date into another format.